apps/cli/src/utils/utils.ts

```diff
@@ -24,6 +24,10 @@ export function repoPath(session: ISession): string {
   return path.resolve(path.join('.', buildPath, 'curvenote'));
 }
 
+export function packageJsonPath(session: ISession): string {
```

Comment:
A little ambiguous name... maybe 
```suggestion
export function webPackageJsonPath(session: ISession): string {
```
or something? That would make `logVersions` slightly more understandable, without reading this function, cli vs web build versions.